doc/website/release-notes/iceoryx-v2-0-0.md

```diff
@@ -6,13 +6,19 @@
 
 **Features:**
 
+- block publisher when subscriber queue full [\#615](https://github.com/eclipse-iceoryx/iceoryx/issues/615)
+- extend chunk header for gateways and recompute [\#711](https://github.com/eclipse-iceoryx/iceoryx/pull/711)
+- extend waitset and listener for member use case and waitset callback for user type [\#707](https://github.com/eclipse-iceoryx/iceoryx/issue/707)
+- Basic Windows 10 support [\#33](https://github.com/eclipse-iceoryx/iceoryx/issues/33)
+- Common puml file for common settings [\#865](https://github.com/eclipse-iceoryx/iceoryx/issues/865)
+- Relocatable Pointers - Version 2.0 [\#926](https://github.com/eclipse-iceoryx/iceoryx/issues/926)
```

Comment:
I'm quite sure the same is true for Windows. The others might really be v2.0 features

Reply:
Yes it's true that windows support was already partly integrated into iceoryx for v1.0. But this issue was finally closed when the named semaphores where integrated after the 1.0 release. Therefore I would see this more here.